test/system/update_patient_info_test.rb

```diff
@@ -216,12 +230,14 @@ class UpdatePatientInfoTest < ApplicationSystemTestCase
         assert has_checked_field? 'Pledge fulfilled'
         assert has_field? 'Procedure date',
                           with: 2.days.from_now.strftime('%Y-%m-%d')
-        assert has_field? 'Date of check',
-                          with: 2.weeks.from_now.strftime('%Y-%m-%d')
-        assert_equal '12', # ??????
+        assert_equal '12',
                      find('#patient_fulfillment_gestation_at_procedure').value
         assert has_field? 'Abortion care $', with: 100
         assert has_field? 'Check #', with: '444-22'
+
+        # There is something deeply, deeply weird about how capybara enters dates.
```

Comment:
Do we want a followup ticket to investigate & fix this?

Reply:
Now in #1314